modules/web/src/common/components/ingressrulelist/template.html

```diff
@@ -46,7 +46,14 @@
         <mat-header-cell *matHeaderCellDef
                          i18n>Path </mat-header-cell>
         <mat-cell *matCellDef="let ingressRuleFlat">
-          {{ !!ingressRuleFlat?.path?.path ? ingressRuleFlat?.path?.path : '-' }}
+          <ng-container *ngIf="ingressRuleFlat.path.path">
+            <ng-container *ngIf="ingressRuleFlat.host">
```

Comment:
Can you merge these two into one `ingressRuleFlat.path.path && ingressRuleFlat.host`?